### ISSUE TITLE
Resend email confirmation 

### ISSUE THREAD

**Issue by jdaudier** [loicginoux/joanne_project#118]
Resend email confirmation link when user tries to log in without having confirmed their account

**Comment by loicginoux** [comment]
agreed, also I think that he email confirmation should have your username written, like this people don't forget what's their username is. I realize that if people forget their username, they can't do anything...
